### DISCUSSION TITLE
Indi and LibXISF and ZSTD

### DISCUSSION THREAD

**Discussion by pcxz** [indilib/indi#2109] [General]
@knro 
I decide to compile, first time, with LibXISF support, only receive this error ( Linux Mint 22 )all other compile good.
...
[ 17%] Building CXX object libs/indibase/CMakeFiles/indidriver_OBJECT.dir/indigpsinterface.cpp.o
/media/Condiviso/Astronomia/indi_ekos_build/Projects/indi-master/libs/indibase/indiccd.cpp: In member function ‘bool INDI::CCD::ExposureCompletePrivate(INDI::CCDChip*)’:
/media/Condiviso/Astronomia/indi_ekos_build/Projects/indi-master/libs/indibase/indiccd.cpp:2471:44: error: ‘CompressionCodecSupported’ is not a member of ‘LibXISF::DataBlock’
 2471 |                     if(LibXISF::DataBlock::CompressionCodecSupported(LibXISF::DataBlock::ZSTD))
      |                                            ^~~~~~~~~~~~~~~~~~~~~~~~~
/media/Condiviso/Astronomia/indi_ekos_build/Projects/indi-master/libs/indibase/indiccd.cpp:2471:90: error: ‘ZSTD’ is not a member of ‘LibXISF::DataBlock’
 2471 |                     if(LibXISF::DataBlock::CompressionCodecSupported(LibXISF::DataBlock::ZSTD))
      |                                                                                          ^~~~
/media/Condiviso/Astronomia/indi_ekos_build/Projects/indi-master/libs/indibase/indiccd.cpp:2472:66: error: ‘ZSTD’ is not a member of ‘LibXISF::DataBlock’
 2472 |                         image.setCompression(LibXISF::DataBlock::ZSTD);
      |                                                                  ^~~~
[ 17%] Building CXX object libs/indibase/CMakeFiles/indidriver_OBJECT.dir/indiweatherinterface.cpp.o
[ 18%] Building CXX object libs/indibase/CMakeFiles/indidriver_OBJECT.dir/indifocuser.cpp.o
make[2]: *** [libs/indibase/CMakeFiles/indidriver_OBJECT.dir/build.make:426: libs/indibase/CMakeFiles/indidriver_OBJECT.dir/indiccd.cpp.o] Errore 1
...

Any info? ..thanks.

**Comment by gnthibault** [comment]
Happened to me on recent Ubuntu distro (24 LTS and likely) If you are building locally:

```
sudo add-apt-repository ppa:mutlaqja/ppa
sudo apt update
sudo apt-get install libxisf-dev
```
